Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -430,4 +428,16 @@ private class DisplayTextField: UITextField {
     fileprivate override var canBecomeFirstResponder: Bool {
         return false
     }
+    
+    override func textRect(forBounds bounds: CGRect) -> CGRect {
```

Comment:
Can you provide some additional context here, a super detailed comment as to why we're overriding this would be helpful, and any potential performance implications too.